src/sardana/macroserver/msmacromanager.py

```diff
@@ -1290,9 +1294,31 @@ def __runXMLMacro(self, xml):
     _runXMLMacro = __runXMLMacro
 
     def runMacro(self, macro_obj):
+
         name = macro_obj._getName()
         desc = macro_obj._getDescription()
         door = self.door
+        self.macro_obj = macro_obj
+
+        if macro_obj.getParentMacro() is None:
+            try:
+                self.logging_onoff = macro_obj.getEnv("LogMacroOnOff")
+            except:
+                self.logging_onoff = False
+            if self.logging_onoff:
+                try:
+                    self.logging_path = macro_obj.getEnv("LogMacroPath")
+                except:
+                    macro_obj.setEnv("LogMacroPath", "/tmp")
+                    self.logging_path = macro_obj.getEnv("LogMacroPath")
```

Comment:
Maybe we could go for variables like: `LogMacroDir` and `LogMacroFile`, similarly to the `ScanDir` and `ScanFile`.

Reply:
No problem, I will change it.